### DISCUSSION TITLE
Using ESM for the next-i18next config

### DISCUSSION THREAD

**Discussion by FleetAdmiralJakob** [i18next/next-i18next#2256] [Q&A]
Can I use ESModules for the config, and if yes how?

**Comment by adrai** [comment]
https://github.com/i18next/next-i18next/issues/2185

**Reply by FleetAdmiralJakob** [reply]
Is there now a doc note somewhere, like it was mentioned in the last comment? I couldn't find it.

**Reply by adrai** [reply]
seems @nick4fake forgot to create a PR
